docs/stdlib/generic.rst

```diff
@@ -270,11 +278,12 @@ Generic
 
     :index: find strpos strstr position array
 
-    A polymorphic function to test if the *haystack* contains the *needle*.
+    Returns :eql:type:`bool` for whether a ``haystack`` set has a ``needle``
+    value.
 
-    When the *haystack* is :eql:type:`str` or :eql:type:`bytes`,
-    return ``true`` if *needle* is contained as a subsequence in it
-    and ``false`` otherwise.
+    When the ``haystack`` is a :eql:type:`str` or :eql:type:`bytes` type,
```

Comment:
```suggestion
    When the *haystack* is a :eql:type:`str` or :eql:type:`bytes` value, this
```
- Use `*` for parameters to remain with our current convention
- haystack may only be a value not a type